### DISCUSSION TITLE
Image Quality not usable when using Versions higher than v15.3.0

### DISCUSSION THREAD

**Discussion by DerGue69** [jomjol/AI-on-the-edge-device#3154] [General]
### The Problem

I have a crisp clear image when using v15.3.0. 
I updated to the latest release v15.7.0 and the Image quality was not usable any more. 
Blurry, resolution was really bad. ROI detection did not work therefore any more....so "it" did not work any more.

So I downgraded back to 15.3.0. With this release the image Quality is perfect.
(I think the problem starts with 15.4.0 ... I tested when it was released and had the same issues)

In Log there is nothing relevant, I posted Info in Log.

### Version

v15.3.0

### Logfile

```shell
Runtime Information
Start time:	19.07.2024 15:41:35
Uptime:	5d 18h 52m 22s
Build Info
Firmware version:	Release: v15.3.0 (Commit: 3fbff0a)
Firmware build time:	22.07.2023 09:42
Web interface version:	Release: v15.3.0 (Commit: 3fbff0a)
Host Info
Hostname:	watermeter
IP Address:	192.168.0.22
WLAN SSID:	UPC Hack Me If You Can
SD Card Info
SD card manufacturer:	SanDisk (ID: 3)
SD card name:	SU02G
SD card size:	1886 MB
SD card sector size:	512 byte
Partition size:	1880 MB
Partition free space:	1825 MB
Partition allocation size:	512 byte
Memory Info
Total Free (Int + Ext):	728519 byte
Ext. RAM - Free:	641908 byte
Ext. RAM - Largest Free Block:	622592 byte
Ext. RAM - Min Free:	512328 byte
Int. RAM - Free:	86611 byte
Int. RAM - Largest Free Block:	81920 byte
Int. RAM - Min Free:	76123 byte
```

### Expected Behavior

Good Image quality when using versions higher than v15.3.0

### Screenshots

Perfect Image Quality when using 15.3.0:
![image](https://github.com/user-attachments/assets/31d71ff9-a87a-4114-a989-1740ca06c6a9)

**Comment by SybexX** [comment]
You always need a counterpart for comparison, but you only put "Perfect Image Quality when using 15.3.0:" image .
In addition, your image "Perfect Image Quality when using 15.3.0:" is also blurry , you would have to readjust the focus.
https://jomjol.github.io/AI-on-the-edge-device-docs/Reference-Image/#focus
